### PR TITLE
fix(memberbox): fix member online status

### LINKED ISSUE #102: Make user's "ONLINE" badge work
Maybe if the logged-in user has activity in last 15/60 minutes, we define it as "online".

See: https://forum.casbin.com/t/63

For reference, see how v2ex does it: https://v2ex.com/member/acorngyl

image

### PR DESCRIPTION
**Pull request by oceanlvr**
Fix: https://github.com/casbin/casbin-forum/issues/102

### CODE REVIEW

**Comment by hsluoyz** [comment]
plz fix based on our conversation.

**Comment by oceanlvr** [comment]
Hi! I just use `commit --force` to this PR, and keep the commit log clean, it shouldn't show it anymore when the user is offline.

File: web/src/main/MemberBox.js [hunk 1]
```diff
@@ -154,7 +154,10 @@ class MemberBox extends React.Component {
               <td width={Setting.PcBrowser ? "73" : "56"} valign="top" align="center">
                 <Avatar username={this.state.member?.id} size={Setting.PcBrowser ? "large" : "middle"} avatar={this.state.member?.avatar} />
                 <div className="sep10" />
-                <strong className="online">ONLINE</strong>
+                {
+                  this.state.member?.onlineStatus ? 
+                  <strong className="online"> {i18next.t("ONLINE")} </strong>:""
```
Comment (hsluoyz) [inline web/src/main/MemberBox.js]: plz format the code.

**Comment by oceanlvr** [comment]
offline:
image

online:
image

**Comment by hsluoyz** [comment]
lgtm

**Pull request merged**